modules/distribution/src/assembly/bin.xml

```diff
@@ -261,6 +261,29 @@
             </includes>
         </fileSet>
 
+
+        <!--copy  web app for UMA 2.0 permission endpoint feature-->
+        <fileSet>
```

Comment:
Why do we need to copy these ? Once touchpoint install these to carbon instance those will be copied batch wise.